Dockerfile-tests

```diff
@@ -1,4 +1,4 @@
-FROM hseeberger/scala-sbt:graalvm-ce-21.0.0-java11_1.5.0_2.13.5
+FROM broadinstitute/scala-baseimage:jdk11-2.13.5-1.4.7
```

Comment:
this probably doesn't need to be changed?

Reply:
Yeah agree.. wanted to see if it fixed tests hanging but seems not so will revert